### PR TITLE
Update guideline_containers.md

### PR DESCRIPTION
**Pull request by The-Debarghya**
Added some security guidelines for container based applications as mentioned in #47

### CODE REVIEW

File: docs/resources/guidelines/guideline_containers.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+### 3. Securing Deployment
+- `Secure the target environment`: You can do this by hardening the underlying host operating system. You can also establish firewall and VPC rules or create special accounts that limit access.
+- `Use an orchestration platform`: These systems usually provide secure API endpoints as well as role-based access control, which can help minimize the risk of unauthorized access.
+- `Use immutable deployments`: This involves creating an instance image during your build steps. Your deployment can then use this image to create new instances.
+
+### 4. Securing Container Runtime
+- Create separate virtual networks for your containers.
+- `Apply the principle of least privilege`: Allow connectivity only between containers that truly need it.
+- `Expose only the ports that serve the application`: Do not expose any other ports, except for SSH. Apply this principle to containers as well as the underlying machines.
```
Comment (ReedOnly) [inline docs/resources/guidelines/guideline_containers.md:21]: You do generally NOT want to expose port 22 (SSH) on any internet facing containers!

**Review by ReedOnly — CHANGES_REQUESTED** [review]
Thanks for contribution! I have added one comment, and will take a closer look over the weekend. Please also add a link to https://equinor.github.io/appsec/snyk/guidelines/4-snyk_container_security/ using internal reference 🙂

File: docs/resources/guidelines/guideline_containers.md (hunk 1 above)
Comment (The-Debarghya) [reply docs/resources/guidelines/guideline_containers.md:21]: Where should I add the link?
At the beginning or end or where I have mentioned Snyk?

**Review by ReedOnly — APPROVED** [review]
It looks good to me! Ready to Merge 😃

**Pull request merged**